Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -39,6 +39,30 @@ Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 # Load all dependent files for execution
 Import-Module ./Utility.ps1 -Force
 
+function Get-SingleFile($files, $pattern)
```

Comment:
we are only checking $files has only one file in it or not, So we should change name of this method as from method name it looks like we are finding files in $files with pattern $pattern. Also we shud not return $dacPacFilePath as it will remain same. we only want to throw if we have multiple files or no file